src/service/history-impl.js

```diff
@@ -751,9 +751,15 @@ export class HistoryBindingVirtual_ {
   push() {
     // Current implementation doesn't wait for response from viewer.
     this.updateStackIndex_(this.stackIndex_ + 1);
+    // TODO(dvoytenko, #8785): cleanup after tracing.
```

Comment:
Got it! Nice `new Error()` log all these info, thought the only info we get would be `reason`.